bare-metal/fedora-coreos/kubernetes/fcc/controller.yaml

```diff
@@ -120,14 +122,15 @@ systemd:
         Type=oneshot
         RemainAfterExit=true
         WorkingDirectory=/opt/bootstrap
+        Environment=KUBELET_IMAGE=quay.io/poseidon/kubelet:v1.20.4
```

Comment:
I'll take the change to bootstrap.service in isolation, since its just matching what's done for other platforms. Please remove the unrelated indentation and other changes and update the description.

Also, the intention of these isn't to support customizing every container image that's used in the cluster. Only to allow selection from the quay and docker.io variants of images built by Poseidon. pause would require additional maintenance to manage and track those versions upstream to keep this in sync and full airgap isn't a feature I'm willing to promise.

Reply:
I think this is resolved.

I guess for now I was keep a separate branch with my changes for air-gap bare-metal installs.